modules/ROOT/pages/user-roles-permissions-and-profiles.adoc

```diff
@@ -61,12 +61,36 @@ All administrators must have the Exchange Viewer permission to import APIs into
 
 |===
 |Permission |Description |Actions
-|Exchange Viewers|Assign this permission to the Organization Administrator or the API Experience Hub Administrator who performs the Salesforce linking process. Users with this permission cannot add new assets, edit asset portal content, or share an asset with another user.
+|Exchange Viewers|Assign this permission to the Organization Administrator or the API Experience Hub Administrator who performs the Salesforce linking process. Users with this permission can't add new assets, edit asset portal content, or share an asset with another user.
 a|
 * View APIs in Exchange
 * Import APIs from Exchange to the portal |
 |===
 
+== User Roles for Configuring SSO
+
+Configuring an identity provider for API Experience Hub includes multiple personas that perform different user actions. 
+
+Depending on your organization’s setup, a user can be assigned one or more of the following roles:
+
+API Experience Hub Administrator:: 
```

Comment:
Consider converting to table to be consistent with Access Management. 
https://confluence.internal.salesforce.com/x/TRRzNQ